### PR TITLE
(PDB-2441) Don't submit TextMessages (only UTF-8) 

### PR DESCRIPTION
**Pull request by rbrw**

### CODE REVIEW

**Comment by rbrw** [comment]
Depends on https://github.com/puppetlabs/puppetdb/pull/1846

**Comment by kbarber** [comment]
So the util code here, I have the same comments about it as per here: https://github.com/puppetlabs/puppetdb/pull/1846

File: src/puppetlabs/puppetdb/mq.clj [hunk 1]
```diff
@@ -205,7 +205,8 @@
 (extend String
   ToJmsMessage
   {:-to-jms-message (fn [x properties session]
-                      (let [msg (.createTextMessage session x)]
+                      (let [msg (.createBytesMessage session)]
+                        (.writeBytes msg (.getBytes x "UTF-8"))
```
Comment (senior) [inline src/puppetlabs/puppetdb/mq.clj]: This call and the createBytesMessage above it use reflection, probably something we should get cleaned up.

**Comment by senior** [comment]
I think the session transacted changes you pulled in are conflicting with Russ' AMQ benchmark change that was just merged

**Pull request merged**